packages/app/src/index.module.scss

```diff
@@ -1,5 +1,5 @@
 // import common sass
-@import '~@repro/common-sass/dist/common.scss';
+@import '~@repro/common-sass/lib/common.scss';
```

Comment:
i'm confused here: there's no `/lib/` folder in this package. Any reason for this change?